plugin/src/py/android_screenshot_tests/pull_screenshots.py

```diff
@@ -52,27 +52,29 @@ def generate_html(dir):
         html.write('<!DOCTYPE html>')
         html.write('<html>')
         html.write('<head>')
+        html.write('<title>Screenshot Test Results</title>')
         html.write('<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>')
         html.write('<script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.3/jquery-ui.min.js"></script>')
         html.write('<script src="default.js"></script>')
         html.write('<link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.3/themes/smoothness/jquery-ui.css" />')
         html.write('<link rel="stylesheet" href="default.css"></head>')
         html.write('<body>')
 
-        html.write('<!-- begin results -->')
-
         for screenshot in sort_screenshots(root.iter('screenshot')):
             alternate = not alternate
+            canonical_name = screenshot.find('name').text
+            last_seperator = canonical_name.rindex('.') + 1
```

Comment:
Hi @xiphirx, this code crashed, when '.' symbol not found on name.
When I can read about this restriction?

Reply:
Interesting... does your test class have no parent package?

Reply:
No, I have a simple test
```
package com.avito.android.design.app.banner_card

class BannerCardTest {

    @Rule
    @JvmField
    val test = BannerCardRule()

    @Test
    fun screenshot__long_message_text_action_and_image() {
        test.bindData(
                BannerCardViewModel(
                        messageText = LONG_MESSAGE,
                        actionText = ACTION_TEXT,
                        image = IMAGE
                )
        )

        waitForImageBeingLoaded()

        test.recordScreenshot("Long_message_text_action_and_image")
    }
}
```
`recordScreenshot` implementation:

```
open fun recordScreenshot(description: String) {
        Component.ui.waitForIdle()

        Screenshot.snap(view)
                .setName(testSuiteName + "__" + description)
                .record()
    }
```

Reply:
As result I have this name: `<name>BannerCardComponent__Long_message_text_action_and_image</name>`

Reply:
Ah! Thanks for the report, I'll push a fix soon